core/domain/question_services.py

```diff
@@ -142,6 +142,26 @@ def get_questions_and_skill_descriptions_by_skill_ids(
     return questions, skill_descriptions, next_cursor
 
 
+def get_questions_by_skill_ids(question_count, skill_ids):
+    """Returns constant number of questions linked to the given skill ids.
+
+    Args:
+        question_count: int. The number of questions to return.
+        skill_ids: list(str). The ID of the skills to which the questions are
```

Comment:
ID --> IDs

are --> should be (since we don't actually have the questions yet)

Reply:
Done.